src/ShaderHelper.h

```diff
@@ -49,13 +49,6 @@ typedef struct {
 

```

Comment:
I would say at this point, just move this struct definition into the MaterialManager class.  This whole file is kind of a crutch anyway, I'll move the Light structure some point this week as well.